pkg/tls/renewer.go

```diff
@@ -20,7 +20,9 @@ const (
 	CAValidityDuration = 365 * 24 * time.Hour
 	// TLSValidityDuration is the valid duration for TLS certificates
 	TLSValidityDuration = 150 * 24 * time.Hour
-	rootCAKey           = "rootCA.crt"
+	// RenewBefore is the renewal time before expiration
+	RenewBefore = 15 * 24 * time.Hour
```

Comment:
Since all the other durations are passed as arguments to the cert manager, should we also pass `RenewBefore` as a variable? That will allow user to configure it while setting up cert manager

Reply:
As far as I remember we agreed to not let users configure this value.